README.rst

```diff
@@ -32,6 +32,29 @@ Which runs all available tests, or run only a specific test::
 
 The above lines are entirely up to the user to modify, and will allow them to choose in which way they want to use the tests.
 
+Running Tests Locally
+#####################
+
+You can start running tests locally by adding the mongoLT (Local Test) hosts with the add-etc-local-hosts.sh bash script as follows::
+
+  $ ./local_setup/add-etc-local-hosts
+  
+Then you can add the required environment variables with the following add-persistent-env-variables.sh bash script::
+
+  $ ./local_setup/add-persistent-env-variables
```

Comment:
Could you recommend here the [local_setup/add-env-variables.sh](https://github.com/kytos-ng/kytos-end-to-end-tests/pull/271/files#diff-0cbf15aec19be11df07a4d8902a55f06c8a9bac93a458b8f9579ded9227b69da) with `sudo` instead? I think we can nuke the `add-persistent-env-variables`. Since we'll need anyway to run with the root user due to `mininet` as you've explained, let's make sure that users will only add this to their `.bashrc` for the root user, using the `/etc/environment` although could be handy it think it can lead to further surprises down the road due to order of how env vars are looked up. See what I mean?